test/functional/config.js

```diff
@@ -3,15 +3,148 @@ var os = require('os');
     sauceLabs: {
-        username:  process.env.SAUCE_USERNAME_FUNCTIONAL,
-        accessKey: process.env.SAUCE_ACCESS_KEY_FUNCTIONAL,
-        build:     process.env.TRAVIS_JOB_ID || '',
-        tags:      [process.env.TRAVIS_BRANCH || 'master'],
-        name:      'testcafe functional tests'
+        username:  process.env.SAUCE_USERNAME_FUNCTIONAL_DESKTOP,
+        accessKey: process.env.SAUCE_ACCESS_KEY_FUNCTIONAL_DESKTOP,
+        jobName:   'functional tests - desktop browsers'
+    },
+
+    browsers: [
+        {
+            platform:    'Windows 10',
+            browserName: 'chrome',
+            alias:       'chrome'
+        },
+        {
+            platform:    'Windows 10',
+            browserName: 'firefox',
+            alias:       'firefox'
+        },
+        {
+            platform:    'OS X 10.10',
+            browserName: 'safari',
+            version:     '8.0',
+            alias:       'safari'
+        },
+        {
+            platform:    'OS X 10.11',
+            browserName: 'chrome',
+            alias:       'chrome-osx'
+        },
+        {
+            platform:    'OS X 10.11',
+            browserName: 'firefox',
+            alias:       'firefox-osx'
+        }
+    ]
+};
+
+testingEnvironments[testingEnvironmentNames.msDesktopBrowsers] = {
+    sauceLabs: {
+        username:  process.env.SAUCE_USERNAME_FUNCTIONAL_MS_DESKTOP,
+        accessKey: process.env.SAUCE_ACCESS_KEY_FUNCTIONAL_MS_DESKTOP,
+        jobName:   'functional tests - ms desktop browsers'
+    },
+
+    browsers: [
+        // TODO: fix hanging browsers (IE10, IE9)
+        {
+            platform:    'Windows 10',
+            browserName: 'microsoftedge',
+            alias:       'edge'
+        },
+        {
+            platform:    'Windows 10',
+            browserName: 'internet explorer',
+            version:     '11.0',
+            alias:       'ie 11'
+        }/*,
+        {
+            platform:    'Windows 8',
+            browserName: 'internet explorer',
+            version:     '10.0',
+            alias:       'ie 10'
+        },
+        {
+            platform:    'Windows 7',
+            browserName: 'internet explorer',
+            version:     '9.0',
+            alias:       'ie 9'
+        }*/
+    ]
+};
+
+testingEnvironments[testingEnvironmentNames.mobileBrowsers] = {
+    sauceLabs: {
+        username:  process.env.SAUCE_USERNAME_FUNCTIONAL_MOBILE,
+        accessKey: process.env.SAUCE_ACCESS_KEY_FUNCTIONAL_MOBILE,
+        jobName:   'functional tests - mobile browsers'
+    },
+
+    browsers: [
+        {
+            platform:    'Linux',
+            browserName: 'android',
+            version:     '5.1',
+            deviceName:  'Android Emulator',
+            alias:       'android'
+        },
+        {
+            platformName:    'iOS',
+            deviceName:      'iPad Retina',
+            platformVersion: '8.4',
+            appiumVersion:   '1.5.3',
```

Comment:
`appiumVersion` is not a required capability, you can just not specify it to get the latest version of Appium.